requirements.txt

```diff
@@ -1,6 +1,7 @@
 civis~=1.6
 requests~=2.18
 click~=6.7
+jupyter-core~=4.6.0
```

Comment:
Due to how sub-dependencies were getting installed, this was causing an import error in Jupyter. This also happens if you try to rebuild the Docker image off master.